profiles/serializer/normalizer.py

```diff
@@ -44,21 +43,29 @@ def normalize_restricted_profile(profile: Profile) -> dict:
 
 @normalize.register(Affiliation)
 def normalize_affiliation(affiliation: Affiliation) -> dict:
+    if affiliation.department is not None:
+        name = [affiliation.department, affiliation.organisation]
+    else:
+        name = [affiliation.organisation]
+
+    if affiliation.address.region is not None:
+        address = [affiliation.address.city, affiliation.address.region,
+                   affiliation.address.country.name]
+        components = {"locality": [affiliation.address.city],
+                      "area": [affiliation.address.region],
+                      "country": affiliation.address.country.name}
+    else:
+        address = [affiliation.address.city, affiliation.address.country.name]
```

Comment:
`address` could be derived from `components`, since it's conceptually a human-readable version of that?

Reply:
updated.